### ISSUE TITLE
configure for frozen fine-tuning

### ISSUE THREAD

**Issue by YuqiHUO** [TengdaHan/MemDPC#5]
hi, thanks for the great work. I just wonder if you can provide the configure of frozen training you mentioned in table 2. How many epochs and what lr did you use? Thank you!

**Comment by chenbiaolong** [comment]
@TengdaHan I would like to know the detail configuration to reproduce the accuracy on ucf101 dataset, any plan to release the train scrips?

**Comment by TengdaHan** [comment]
Sorry for the delay, I just updated some evaluation code for your reference. See here [https://github.com/TengdaHan/MemDPC/tree/master/eval](https://github.com/TengdaHan/MemDPC/tree/master/eval)

Let me know if there are any problems.

**Comment by YuqiHUO** [comment]
Why do you use dropout = 0.9 in MemDPC but 0.5 in your previous work DPC? I found that most video self-supervised work use dropout=0.5, did you find something interesting?

**Comment by TengdaHan** [comment]
I found dropout = 0.9 for finetuning entire network gives much better classification accuracy (experimented dropout = 0.5, 0.7, 0.9). We re-evaluate DPC weights under the exact same setting in MemDPC paper Table 1, and it also gets better results. 

Just curious, can you let me know which work(s) uses dropout = 0.5? I found many papers didn't mention this detail.

**Comment by YuqiHUO** [comment]
thanks, https://arxiv.org/pdf/2003.02692 (PSP) https://arxiv.org/pdf/2008.03800 (CVRL), CMC, VCOP used dropout = 0.5. I have tried 0.5 vs 0.9 in my code, finding that 0.5 performs way much better than 0.9. So I'm very curious about your config.
Maybe is because you use final_bn and other methods (including mine) didn't use.

**Comment by TengdaHan** [comment]
On my side, finetuning without the final_bn still works better with 0.9 dropout. I think it's because UCF101 and HMDB51 are too small and very easy to overfit the training set. The effect of large dropout will show up when you train the classifier for longer. 
Thank you for letting me know.
